### ISSUE TITLE
Kpt fn eval help text is out of date

### ISSUE THREAD

**Issue by mengqiy** [GoogleContainerTools/kpt#1980]
Currently it still shown some `kpt fn run` stuff:
```
$ kpt fn eval --help

  kpt fn run DIR [flags]

If the container exits with non-zero status code, run will fail and print the
container `STDERR`.

  DIR:
    Path to a package directory.  Defaults to stdin if unspecified.

Usage:
  kpt fn eval [DIR | -] [flags]

Examples:

  # read the Resources from DIR, provide them to a container my-fun as input,
  # write my-fn output back to DIR
  kpt fn run DIR/ --image gcr.io/example.com/my-fn

  # provide the my-fn with an input ConfigMap containing `data: {foo: bar}`
  kpt fn run DIR/ --image gcr.io/example.com/my-fn:v1.0.0 -- foo=bar

  # run the functions in FUNCTIONS_DIR against the Resources in DIR
  kpt fn run DIR/ --fn-path FUNCTIONS_DIR/


  # discover functions in DIR and run them against Resource in DIR.
  # functions may be scoped to a subset of Resources -- see `kpt help fn run`
  kpt fn run DIR/
```

**Comment by mengqiy** [comment]
cc: @droot @Shell32-Natsu

**Comment by droot** [comment]
Yes, I am working on it, will send a PR soon.

**Comment by droot** [comment]
This is addressed with #1981
